public/img/icons/activity-icons/LaborMarket.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can we do kebab-case for all the icon file names?

Reply:
oh I see the trouble with that in `activity-avatar.tsx`. I think I would create a new attribute on the activity schema for "icon-type" or something. bleh.